pkg/apis/internal.acorn.io/v1/appimage.go

```diff
@@ -28,7 +28,8 @@ type VCS struct {
 	// Modified a true value indicates the build contained modified files according to git
 	Modified bool `json:"modified,omitempty"`
 	// Untracked a true value indicates the build contained untracked files according to git
-	Untracked bool `json:"untracked,omitempty"`
+	Untracked bool   `json:"untracked,omitempty"`
+	Acornfile string `json:"acornfile,omitempty"`
```

Comment:
Please add a godoc comment above this field, describing its purpose (this shows up in the OpenAPI schema and is also useful for future squirrels)